Mathlib/Util/CountHeartbeats.lean

```diff
@@ -62,15 +60,15 @@ def logVariation {m} [Monad m] [MonadLog m] [AddMessageContext m] [MonadOptions
   -- convert `[min, max, stddev]` to user-facing heartbeats
   logInfo s!"Min: {min / 1000} Max: {max / 1000} StdDev: {stddev / 10}%"
 
-/-- Count the heartbeats used by a tactic, e.g.: `count_heartbeats simp`. -/
-elab "count_heartbeats " tac:tacticSeq : tactic => do
+/-- Count the heartbeats used by a tactic, e.g.: `#count_heartbeats simp`. -/
+elab "#count_heartbeats " tac:tacticSeq : tactic => do
```

Comment:
Would it be possible to keep the old syntax as e.g. a deprecated macro? I think this may be confusing otherwise if someone doesn't notice this change

Reply:
I just added a deprecation, thanks!